core/src/main/scala/org/apache/spark/storage/DiskBlockManager.scala

```diff
@@ -133,7 +133,6 @@ private[spark] class DiskBlockManager(blockManager: BlockManager, conf: SparkCon
     Utils.getConfiguredLocalDirs(conf).flatMap { rootDir =>
       try {
         val localDir = Utils.createDirectory(rootDir, "blockmgr")
-        Utils.chmod700(localDir)
```

Comment:
Actually standalone seems to be missing the chmod call; I'll just add that to this change.